cmd/kubeadm/.import-restrictions

```diff
@@ -77,7 +77,7 @@
 				"k8s.io/kubernetes/pkg/util/conntrack",
 				"k8s.io/kubernetes/pkg/util/hash",
 				"k8s.io/kubernetes/pkg/util/iptables",
-				"k8s.io/kubernetes/pkg/util/metrics",
+				"k8s.io/component-base/metrics/prometheus/ratelimiter",
```

Comment:
I don't think this makes sense in a list of imports that match `"SelectorRegexp": "k8s[.]io/kubernetes/pkg",`... can probably just drop this